subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc

```diff
@@ -134,6 +134,10 @@ When set to `all` or `full`, a stacktrace is present in the output for all excep
 Using `full` doesn't truncate the stacktrace, which leads to a much more verbose output.
 _Default is `internal`._
 
+`org.gradle.welcome-message=(true,false)`::
+When set to `false`, Gradle won't generate welcome message.
+_Enabled by default and welcome message is shown on first execution of each new version of Gradle._
```

Comment:
I think we should adjust the key and the values to the other Gradle properties. How about this: 

```suggestion
`org.gradle.welcome=(never,once)::
Controls whether Gradle should print a welcome message. If set to _never_ then the welcome message will be suppressed. If set to _once_ then the message is printed once for each new version of Gradle.
_Default is `once`._
```

Reply:
Nonetheless, the PR looks good. Thank you @yasmans for taking the time and implementing this feature!